classes/plugininfo/factor.php

```diff
@@ -138,8 +138,8 @@ public static function get_factor_actions() {
         $actions[] = 'add';
         $actions[] = 'revoke';
         $actions[] = 'enable';
```

Comment:
1) looks like duplicate 'revoke's
2) is 'enable' wrong?

Reply:
Revoke - for factor instance (created by user). 
Disable/enable for factor type (admin settings)